integration/src/main/java/scripts/utils/ClientTestUtils.java

```diff
@@ -388,4 +368,16 @@ public static <T> T assertPresent(Optional<T> optional, @Nullable String message
   public static <T> T assertPresent(Optional<T> optional) {
     return assertPresent(optional, null);
   }
+
+  @FunctionalInterface
+  public interface SupplierWithException<T> {
+    T get() throws Exception;
+  }
+
+  /** An interface for an arbitrary workspace operation that throws an {@link ApiException}. */
+  @FunctionalInterface
+  public interface WorkspaceOperation<T> {
+
```

Comment:
nit: spacing is different in this function vs the one above. I'm curious why spotless is fine with both.

Reply:
I think the standard is two space tabs. I had previously set mine, but switched IntelliJ versions and didn't reset. It seems we don't have spotless insisting on two space tabs.